model/project.go

```diff
@@ -37,6 +37,7 @@ const (
 	waterfallTasksQueryMaxTime = 90 * time.Second
 )
 
+// Project represents the fully hydrated project configuration
```

Comment:
What does "fully hydrated" mean in this sense?

Reply:
You can read about data hydration [here](https://stackoverflow.com/a/20787106). Basically, we're starting with a `ParserProject` (without tags expanded to all the matching tasks, so it's only partially hydrated), and then it gets translated to a `Project` (with tags expanded, so it's fully hydrated) so it's actually usable.